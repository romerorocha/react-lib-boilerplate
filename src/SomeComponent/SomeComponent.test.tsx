import { render } from '@testing-library/react'
import { SomeComponent } from '.'

describe('<SomeComponent />', () => {
  it('should render default title and body', () => {
    const { getByRole, getByText, container } = render(<SomeComponent />)

    getByRole('heading', { name: /hello!/i })
    getByText(/hello, world!/i)

    expect(container.firstChild).toMatchSnapshot()
  })
})
