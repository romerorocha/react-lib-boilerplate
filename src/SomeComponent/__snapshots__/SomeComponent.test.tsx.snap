// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SomeComponent /> should render default title and body 1`] = `
<div>
  <h1>
    Hello!
  </h1>
  <p>
    Hello, world!
  </p>
</div>
`;
